src/components/member-page/member-data.js

```diff
@@ -88,7 +85,7 @@ const MemberData = ({
 }
 
 MemberData.propTypes = {
-  role: PropTypes.oneOf(Object.values(MEMBER_ROLE)),
+  role: PropTypes.object,
```

Comment:
could we specify the shape of role object

Reply:
fixed